process/request_resources.md

```diff
@@ -41,16 +42,14 @@ Limited, time-bound maintainer stipends for specified project improvements may b
 
 ### Security audits and improvements
 
-Qualifying projects may request funding for security audits and other services offered by third party security professionals. Before creating a request for proposals, projects should confirm funding availability with the OpenSSF Budget Committee. After receiving proposals, projects should request their selected proposal be funded through the Budget Committee. Projects who need assistance creating an RFP should contact the OpenSSF Planning Committee. RFPs must be posted in a public channel(s) (GitHub issue, mailing list). OpenSSF member companies may bid on RFPs. 
+Qualifying projects may request funding for security audits and other services offered by third party security professionals. Before creating a request for proposals, projects should confirm funding availability with the OpenSSF Budget Committee. After receiving proposals, projects should request their selected proposal be funded through the Budget Committee. Projects that need assistance creating an RFP should contact the OpenSSF Planning Committee. RFPs must be posted in a public channel(s) (GitHub issue, mailing list). OpenSSF member companies may bid on RFPs.
 
 ## Other
 
 ### Code of Conduct support
 
 All projects must follow the [OpenSSF Code of Conduct](https://openssf.org/community/code-of-conduct/), or, if a project has its own Code of Conduct, it must be reviewed and approved by the TAC. 
 
-If projects receive reports of conduct violations, they should report them to conduct@openssf.org.
```

Comment:
Should we keep this and maybe just point to operations@ ?

Reply:
No, Brian pointed out that the paragraph above suffices. The CoC tells people what to do.

Reply:
SGTM, resolving this.